components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadNotification.kt

```diff
@@ -68,16 +71,19 @@ internal object DownloadNotification {
     /**
      * Build the notification to be displayed while the download service is paused.
      */
-    fun createPausedDownloadNotification(context: Context, downloadState: DownloadState): Notification {
+    fun createPausedDownloadNotification(context: Context, downloadJobState: DownloadJobState): Notification {
         val channelId = ensureChannelExists(context)
 
+        val downloadState = downloadJobState.state
         return NotificationCompat.Builder(context, channelId)
             .setSmallIcon(R.drawable.mozac_feature_download_ic_download)
             .setContentTitle(downloadState.fileName)
             .setContentText(context.getString(R.string.mozac_feature_downloads_paused_notification_text))
             .setColor(ContextCompat.getColor(context, R.color.mozac_feature_downloads_notification))
             .setCategory(NotificationCompat.CATEGORY_PROGRESS)
             .setOngoing(true)
+            .setWhen(downloadJobState.createdTime)
```

Comment:
I am wondering if this should be a separate timestamp i.e. the time the notification happened ( == the time the download was paused, not created)?

As this is a different notification than the in progress one.

Reply:
It really depends on how we want the UI to react, if we use a different timestamp, the notification will change positions when we click the resume button. If we use the same the notification will remain in the same position. I'm not sure what should be the correct behaviour?

Reply:
👍